exonum-java-binding/launcher-plugin/exonum_java_runtime_plugin/plugin.py

```diff
@@ -17,7 +17,7 @@
 from exonum_launcher.runtimes.runtime import RuntimeSpecLoader
 
 try:
-    from .proto import service_runtime_pb2
+    from .proto import deploy_arguments_pb2
```

Comment:
What does 'pb2' mean? We use proto3, is everything compatible?

Reply:
Honestly I didn't know, but it looks okay https://github.com/grpc/grpc/issues/15444#issuecomment-396442980